### DISCUSSION TITLE
nativescript-ui-*

### DISCUSSION THREAD

**Discussion by kefahB** [NativeScript/NativeScript#10173] [General]
Hi,

Is the NS team or Community that maintain the `nativescript-ui` ? it is always frustrating when we have an issues with those ui plugins, there is [this link](https://github.com/ProgressNS/nativescript-ui-feedback/issues) pointing to the repo but no body respond! 

The `nativescript-ui` is a well used plugin and I think it should take more care.

For example `RadListvVew` has a several issue with grouping and sorting functions! loadOnDemande not working with grouping functions on IOS...

@NathanWalker what we can do ?

**Comment by NathanWalker** [comment]
Thanks for bringing this up. We do generally as a community want to move away from them (nativescript-community/ui-collectionview is a good successor as it’s extensible and only missing group/sort/swipe at moment to be feature compliant but already works better than RLV). Similarly the community drawer plugin is also already better than RadSideDrawer.

However if you want to help on these in meantime please DM me on Discord as they have to remain private due to legality.

**Reply by kefahB** [reply]
Can you give me your username on Discord ?
